### PR TITLE
Added kaishi65

### PR DESCRIPTION
**Pull request by powerlemming**
This PR aims to add compatibility with the kaishi65 keyboard.

Feel free to let me know if my code needs any change :)

### CODE REVIEW

File: keymapviz/keyboards/kaishi65/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+     * |----------------------------------------------------------------|
+     * | {}  | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}  | {} |
+     * |----------------------------------------------------------------|
+     * |  {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
+     * |----------------------------------------------------------------|
+     * |   {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}  | {}| {} |
+     * |----------------------------------------------------------------|
+     * | {} | {} | {} |          {}           | {}| {}| {} | {}| {}| {} |
+     * `----------------------------------------------------------------'                     
+     * generated by [keymapviz]
```
Comment (yskoht) [inline keymapviz/keyboards/kaishi65/__init__.py:16]: Please add c language block comment end `*/`.
```suggestion
     * generated by [keymapviz]
     */
```

**Comment by powerlemming** [comment]
I think I did all the required changes. 
Sorry if I messed up with the commits as I am a total noob  when it comes to working  with Git CLI...
Please let me know if you need me to change something !

File: keymapviz/keyboards/kaishi65/__init__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+keymap_keyword = '(?:LAYOUT_65_ansi)'
+layout_editor_json = 'keyboards/kaishi65/layout_editor.json'
+
+ascii_art = '''
+    /*,----------------------------------------------------------------.
```
Comment (yskoht) [inline keymapviz/keyboards/kaishi65/__init__.py:5]: Okey. keymapviz outputs follows, 

```
/* ,----------------------------------------------------------------. # <-- First line (has no spaces)
     * |ES |1  | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | MI| EQ|  BS   | GR | # <-- 2nd line  (has spaces)
     * |----------------------------------------------------------------| # <- ... (has spaces)
     * | TA  | Q | W | E | R | T | Y | U | I | O | P | LB| RB| BS  | DE | # <- ... (has spaces)
     * |----------------------------------------------------------------| # <- ... (has spaces)
     * |  CA   | A | S | D | F | G | H | J | K | L | SC| QU|  EN   | PG | # <- ... (has spaces)
     * |----------------------------------------------------------------| # <- ... (has spaces)
     * |   LS   | Z | X | C | V | B | N | M | CO| DO| SL|  RS  | UP| PG | # <- ... (has spaces)
     * |----------------------------------------------------------------| # <- ... (has spaces)
     * | LC | LG | LA |          SP           | RA| MO| RC | LE| DO| RG | # <- ... (has spaces)
     * `----------------------------------------------------------------' # <- ... (has spaces)
     * generated by [keymapviz] # <- ... (has spaces)
     */ # <- last line (has spaces)
```

First line has no spaces, but 2nd ~ last lines have spaces at the head of line.

I think it should be outputted like follows. The positions of `*` are aligned.

```
/* ,----------------------------------------------------------------.
 * |ES |1  | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | MI| EQ|  BS   | GR |
 * |----------------------------------------------------------------|
 * | TA  | Q | W | E | R | T | Y | U | I | O | P | LB| RB| BS  | DE |
 * |----------------------------------------------------------------|
 * |  CA   | A | S | D | F | G | H | J | K | L | SC| QU|  EN   | PG |
 * |----------------------------------------------------------------|
 * |   LS   | Z | X | C | V | B | N | M | CO| DO| SL|  RS  | UP| PG |
 * |----------------------------------------------------------------|
 * | LC | LG | LA |          SP           | RA| MO| RC | LE| DO| RG |
 * `----------------------------------------------------------------'
 * generated by [keymapviz]
 */
```

So, I think your source code should be fixed like follows, 

```
ascii_art = '''
/* ,----------------------------------------------------------------.
 * |{} |{} | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
 * |----------------------------------------------------------------|
 * | {}  | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}  | {} |
 * |----------------------------------------------------------------|
 * |  {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
 * |----------------------------------------------------------------|
 * |   {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}  | {}| {} |
 * |----------------------------------------------------------------|
 * | {} | {} | {} |          {}           | {}| {}| {} | {}| {}| {} |
 * `----------------------------------------------------------------'
 * generated by [keymapviz]
 */
```

Diff is here.
```diff
 ascii_art = '''
-    /* ,----------------------------------------------------------------.
+/* ,----------------------------------------------------------------.
-     * |{} |{} | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
+ * |{} |{} | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
-     * |----------------------------------------------------------------|
+ * |----------------------------------------------------------------|
-     * | {}  | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}  | {} |
+ * | {}  | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}  | {} |
-     * |----------------------------------------------------------------|
+ * |----------------------------------------------------------------|
-     * |  {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
+ * |  {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
-     * |----------------------------------------------------------------|
+ * |----------------------------------------------------------------|
-     * |   {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}  | {}| {} |
+ * |   {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}  | {}| {} |
-     * |----------------------------------------------------------------|
+ * |----------------------------------------------------------------|
-     * | {} | {} | {} |          {}           | {}| {}| {} | {}| {}| {} |
+ * | {} | {} | {} |          {}           | {}| {}| {} | {}| {}| {} |
-     * `----------------------------------------------------------------'                     
+ * `----------------------------------------------------------------'                     
-     * generated by [keymapviz]
+ * generated by [keymapviz]
-     */
+ */

'''
```

**Comment by yskoht** [comment]
It looks good to me! I will release this PR at version 1.2.0. Thanks.

**Pull request merged**